aspnetcore/migration/60-70.md

```diff
@@ -71,17 +71,44 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 </ItemGroup>
 ```
 
-<!--
 ## Blazor
 
-After following the guidance earlier in this article to update an app to 7.0, adopt specific features by following the links in <xref:aspnetcore-7.0#blazor>.
+### Adopt .NET 7 features
+
+After following the guidance earlier in this article to update an app to 7.0, adopt specific features by following the links in <xref:aspnetcore-7#blazor>.
 
-To adopt all of the [new 6.0 features for Blazor apps](xref:aspnetcore-7.0#blazor), we recommend the following process:
+To adopt all of the [new 7.0 features for Blazor apps](xref:aspnetcore-7#blazor), we recommend the following process:
 
 * Create a new 7.0 Blazor project from one of the Blazor project templates. For more information, see <xref:blazor/tooling>.
 * Move the app's components and code to the 7.0 app making modifications to adopt the new 7.0 features.
+
+<!--
+
+    IN PROGRESS
+
+    Waiting for the PR to merge.
+
+### Migrate unmarshalled JavaScript interop
+
+Unmarshalled interop using the <xref:Microsoft.JSInterop.IJSUnmarshalledRuntime> interface is obsolete and should be replaced with .NET JavaScript `[JSImport]`/`[JSExport]` interop.
+
+For more information, see <xref:blazor/js-interop/import-export-interop>.
+
 -->
 
+### .NET WebAssembly build tools for .NET 6 projects
+
+You can now use the .NET WebAssembly build tools with a .NET 6 project when working with the .NET 7 SDK. The new `wasm-tools-net6` workload includes the .NET WebAssembly build tools for .NET 6 projects so that they can be used with the .NET 7 SDK. The existing `wasm-tools` workload installs the .NET WebAssembly build tools for .NET 7 projects. However, the .NET 7 version of the .NET WebAssembly build tools are incompatible with existing projects built with .NET 6. Projects using the .NET WebAssembly build tools that need to support both .NET 6 and .NET 7 must use multi-targeting.
+
+### Improvements to the .NET WebAssembly build tools
```

Comment:
I meant this section should move to the what's new doc. These are new features in the wasm-tools workload for .NET 7.